website/usage/_data.json

```
@@ -92,7 +92,7 @@
             "Dependency Parse": "dependency-parse",
             "Named Entities": "named-entities",
             "Tokenization": "tokenization",
-            "Sentence Segmentation": "sbd",
+            "Sentence Segmentation": "sentencizer",
```

I probably wouldn't change the link anchor names – otherwise, we'd have to add a bunch of server-side rewriting rules to make sure existing links to those docs sections don't break. I think it's okay if link anchors are short and more cryptic, so for simplicity, I'd keep them all as `sbd`:

```suggestion
            "Sentence Segmentation": "sbd",
```